.pre-commit-config.yaml

```diff
@@ -0,0 +1,30 @@
+# Copyright 2023 Iguazio
```

Comment:
```suggestion
# Copyright 2024 Iguazio
```

Reply:
This is the suggested template:
https://github.com/mlrun/mlrun/blob/ba399e9f487b769324d9f9e98a8a047ea9c39a74/.github/workflows/ci.yaml#L68

And I think that it should be changed for all the files 🤔
I'll try updating it here, but it's a wider issue

Reply:
Ok, it worked 🙂
96bb5d0

Reply:
It's the year the file was added. You don't need to update all the existing files.